projects/frontend/data-pipelines/gui/README.md

```diff
@@ -5,7 +5,7 @@ The gui part of data-pipelines contains:
   * **[Data Pipelines](projects/data-pipelines/README.md)** Library that holds UI Components for managing Data Pipeline API.
   * **[UI](projects/ui/src)** Default UI Wrapper Implementation of the Data Pipeline library
 
-The wrapper ([ui](projects/ui/src)) provides standard OAuth2 authorization flow, configuration of which could be found in [auth.ts](projects/ui/src/app/auth.ts)
+The wrapper ([ui](projects/ui/src)) provides standard OAuth2 authorization flow, configuration of which could be found in [appConfig.json](projects/ui/src/assets/data/appConfig.json)
```

Comment:
It's not immediately clear how we disable the authorization using the config. Can you please add it to the docs?

Reply:
It's not yet implemented. In the scope of this PR, is making configurable what was pre-existing, without the complexity (yet) of adding new stuff. We will add the additional configs in another PR

Reply:
Is this scoped anywhere?